File: PRAKTIKUM 1/Bahasa C/PRAK104-2510817120014-NailaAtresiaRahma.c
#include <stdio.h>
int main (){
    float a = 400000;
    float b = 350000;
    float adiskon = 13;
    float bdiskon = 21;
    float akurang = a * adiskon/100;
    float bkurang = b * bdiskon / 100;
    float akhir1 = a - akurang;
    float akhir2 = b - bkurang;
    
    printf("Harga sepatu A adalah %.0f\n", a);
    printf("Harga sepatu B adalah %.0f\n", b);
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %.0f\n", akhir1);
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %.0f\n", akhir2);
    
return 0;
}

File: PRAKTIKUM 1/Bahasa Python/PRAK104-2510817120014-NailaAtresiaRahma.py
a = 400000
b = 350000
adiskon = 13
bdiskon = 21
akurang = a * adiskon/100
bkurang = b * bdiskon / 100
akhir1 = a - akurang
akhir2 = b - bkurang

print(f"Harga sepatu A adalah {a}")
print(f"Harga sepatu B adalah {b}")
print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {akhir1:.0f}")
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {akhir2:.0f}")